package NYT.NTableServer.NProto;

////////////////////////////////////////////////////////////////////////////////

message TMsgSetTableChunk
{
    required bytes TransactionId = 1;
    required bytes NodeId = 2;
    required bytes ChunkId = 3;
}

////////////////////////////////////////////////////////////////////////////////
